src/nav/AppContainer.js

```diff
@@ -66,6 +67,8 @@ class AppContainer extends PureComponent {
       actions.fetchEssentialInitialData();
       actions.fetchRestOfInitialData(pushToken);
       actions.fetchEvents();
+      actions.fetchUsersStatus(); // do a initial fetch
+      setInterval(() => actions.fetchUsersStatus(), config.activePingInterval);
```

Comment:
Cool, i tested this on a simulator and found the status send repeatedly, maybe not true for a device...